### PR TITLE
Added prime number code in cpp

### PR DESCRIPTION
**Pull request by soukarsha122**
Issue #157 .
I added a code in C++ to check if a number is prime or not. The code even checks if the number inputted is integer or not as only positive integral numbers are in the set  of prime numbers. This is a small contribution on behalf of me for GSSOC'21.

### CODE REVIEW

File: prime number.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  #include <iostream>
+  using namespace std;
+  int main()
+   {
+    float N;
+  	cin>>N;  // Taking input the number from the user
+  	
+  	int factor=0; // Variable to store number of factors of the inputted number
+  	
+  	int x=(int)N; // Variable to store the integer part of the inputted number
```
Comment (rudrakshi99) [inline prime number.cpp:18]: No need to typecast N. Just simply input integer value.

File: prime number.cpp [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+   Std    output: 1.4 is not a prime number
+   
+   Sample input: 1
+   Std    output: 1 is not a prime number
+   
+   Sample input: -4
+   Std    output: -4 is not a prime number
+   
+   Sample input: 11
+   Std    output: 11 is a prime number
```
Comment (rudrakshi99) [inline prime number.cpp:58]: Add time and space complexity at the end.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by rudrakshi99** [comment]
Also, add it to a proper folder.

**Comment by soukarsha122** [comment]
@rudrakshi99 can you please help me by saying how can i add my file to the C++ folder . Which git command do i have to use ? I am a beginner in git ,so I don't know all git commands .

**Comment by soukarsha122** [comment]
@rudrakshi99 @plazzy99 please review my code and tell me if any more changes needs to be done. I am very new to github and i faced difficulties while updating readme . Please tell me if anything is wrong.

File: Algorithm/Number_theory/readme.md [hunk 3]
```diff
@@ -36,6 +36,5 @@
 <hr>
 
 
-# Questions related to Number Theory
-
```
Comment (plazzy99) [inline Algorithm/Number_theory/readme.md:40]: you are strictly not allowed to delete any line written before.
Revert changes asap else your pr will be closed

**Review by plazzy99 — CHANGES_REQUESTED** [review]

**Comment by soukarsha122** [comment]
Please see now @plazzy99 . I have done as requested. Please tell me if I am wrong . Please tell me what further changes I need to do. Please merge this pr.

**Comment by soukarsha122** [comment]
@plazzy99 @rudrakshi99 @tarun26091999 please review my changes

**Comment by soukarsha122** [comment]
@raksha009  @shreyanspoddar please review my pr please merge it

File: Algorithm/Number_theory/readme.md [hunk 4]
```diff
@@ -36,7 +36,7 @@
 <hr>
 
 
-## Questions related to Number Theory
+# Questions related to Number Theory
```
Comment (rudrakshi99) [inline Algorithm/Number_theory/readme.md:39]: add one `#` also.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by soukarsha122** [comment]
@rudrakshi99 I have done the changes as requested.  Please tell me if any further changes needs to be done..
Please merge my pr..Thank you

**Review by plazzy99 — APPROVED** [review]

**Pull request merged**